styles/index.css

```diff
@@ -0,0 +1,177 @@
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  height: 100vh;
+}
+
+header {
+  margin: 1rem 1rem 1.3rem 1rem;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border: 2px solid #000;
+  padding: 0.5rem 0.5rem 0.5rem 1.7rem;
+}
+
+.logo {
+  font-size: 1.3rem;
+  font-weight: bold;
+}
+
+.menu {
+  display: flex;
+}
+
+.menu-item {
+  padding: 0.5rem 1rem;
+}
+
+.menu .menu-item:nth-child(2) {
+  border-left: 2px solid #000;
+  border-right: 2px solid #000;
+}
+
+.link-item {
+  color: #000;
+  font-weight: bold;
+  text-decoration: none;
+  font-size: 1.3rem;
+}
+
+.visited {
+  color: #3c99dc;
+}
+
+.date {
+  margin-top: 0.5rem;
+  font-size: 1.1rem;
+  float: right;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 0 1rem 5.5rem 1rem;
+  overflow-y: auto;
+}
+
+.book-list {
+  display: none;
+  flex-direction: column;
+  align-items: center;
+  width: 50%;
+}
+
+.main-title {
+  font-size: 2rem;
+  font-weight: bold;
+  margin-bottom: 1.2rem;
+}
+
+.list-cont {
+  display: block;
+  border: 2px solid #000;
+  align-self: stretch;
+  border-radius: 3px;
+}
+
+.list-cont div:nth-child(odd) {
+  background-color: #b8b8b8;
+}
+
+.book-cont {
+  display: flex;
+  justify-content: space-between;
+  padding: 0.5rem 1rem;
+  font-size: 0.5rem;
+  box-sizing: border-box;
+  align-items: center;
+}
+
+.book-info {
+  font-size: 1rem;
+  font-weight: bold;
+}
+
+.rmbtn {
+  box-shadow: 4px 5px #000;
+  border: 1px solid #000;
+  font-size: 1rem;
+}
+
+.rmbtn:hover {
+  cursor: pointer;
+}
+
+.form {
+  display: none;
+  flex-direction: column;
+  align-items: center;
```

Comment:
- [ ] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
This is just an example if you have same CSS rules please combine them.
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)